src/partners/moonpay.ts

```diff
@@ -126,3 +146,41 @@ export function processMoonpayTx(rawTx: unknown): StandardTx {
   }
   return standardTx
 }
+
+function getFiatPaymentType(tx: MoonpayTx): FiatPaymentType | null {
+  switch (tx.paymentMethod) {
+    case undefined:
```

Comment:
when does undefined happen?

Reply:
I'll find a document with no paymentMethod in the rawTx as a case in point and we'll yell at Moonpay.

Reply:
```json
{
   "selector": {
      "rawTx.paymentMethod": {
         "$exists": false
      }
   }
}
```

That's the selector to use on the `edge_moonpay` partition to find documents without the paymentMethod